README.md

```diff
@@ -29,15 +31,23 @@ D4M is slow. Primarily because of its osxfs/grpcfuse sharing filesystem. On the
 There are various ways to code in this setup. I have outlined my decisions in a [thorough article](https://medium.com/homullus/docker-for-mac-performance-diy-d4m-e4232ca8b671).
 
 ### Naked
-You can just go oldschool and keep coding on your local machine. Use your editor of choice. All the files will be sync-ed over to the containers at nearly native speeds.
+Files - Host&VM; Editor - Host;
```

Comment:
This looks to me almost like a broken HTML entity at a glance 😄 

Maybe you can improve formatting of this block using a table, to make it more obvious what are you trying to say?

Reply:
@markomitranic ☝️

Reply:
Hope this makes it better @borisceranic :)

Reply:
Fantastic! You have unleashed your inner designer 😄